docs/decisions/0002-proctoring-email-template-overrides.rst

```diff
@@ -0,0 +1,48 @@
+Proctoring Email Template Overrides
+-----------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+edX sends a learner email(s) with the status of their proctored exam attempt after the learner
```

Comment:
`edX sends learner email(s)`

Reply:
I think what I meant was, "edX sends email(s) to a learner...". I think that's more clear, so I'll use that.